app/gui2/src/components/NodeSpan.vue

```diff
@@ -65,14 +65,15 @@ watch(exprRect, (rect) => {
     style="{ transform }"
     :data-span-id="props.span.id"
     :data-span-start="props.offset"
-    ><template v-if="props.span.children.length > 0"
-      ><NodeSpan
+    ><template v-if="props.span.children.length > 0">
+      <NodeSpan
```

Comment:
Be careful about that formatting. This change introduces another text node into the DOM. Because the node content is currently `contenteditable`, it is actually significant. The "weird" html formatting here was done on purpose to avoid that. At least linters understand that.

Reply:
ah, that makes sense